xwiki-platform-core/xwiki-platform-blog/xwiki-platform-blog-ui/src/main/resources/Blog/BlogParameters.xml

```diff
@@ -74,6 +74,6 @@
  * @param $imgName The name of the icon to use (without the extension).
  * @param alt The alternate text that should be displayed when images are not viewable.
  *#
-#macro(toolImage $imgName $alt)[[image:path:$xwiki.getSkinFile("icons/silk/${imgName}.png")||class="icon-manage" alt="${alt}"]]#end
+#macro(toolImage $imgName $alt)(% class="glyphicon glyphicon-${imgName} icon-manage" spam="${alt}"%) #end
```

Comment:
What happened is that we had the Silk icon set before. Then we added other icon sets, see http://design.xwiki.org/xwiki/bin/view/Proposal/XWiki+Icon+Set. But we must not hardcode any set used as otherwise when you switch skin you'll get the wrong icon set (as was proved by the fact that you sent this PR!). The problem with your commit is that if we apply it, it'll work only in Flamingo.

This is why we've come up with a mapping, see http://extensions.xwiki.org/xwiki/bin/view/Extension/Icon+Theme+Application#HForDevelopers which explains more.